packages/metamask/src/index.ts

```diff
@@ -63,6 +63,12 @@ export class MetaMask extends Connector {
         })
 
         this.provider.on('disconnect', (error: ProviderRpcError): void => {
+          // 1013 indicates that MetaMask is attempting to reestablish the connection
+          // https://github.com/MetaMask/providers/releases
```

Comment:
Please link more directly. Either to the actual release with this documentation (https://github.com/MetaMask/providers/releases/tag/v8.0.0), or to the code that implements it.

Reply:
Thank you! Updated the link